cLIMS/organization/importSeqFiles.py

```diff
@@ -11,78 +11,79 @@ def importSeqFiles(request,prj_pk):
-                md5sum=True
-                md5sumIndx=v.index("md5sum")
-                orderList.append("md5sum")
-            if("sha256sum" in v):
-                sha256=True
-                sha256Indx=v.index("sha256sum")
-                orderList.append("sha256sum")
+        runName=expName=path=""
+        md5Sum=sha256sum=BackupPath=""
+        if(v[0]!=""):
+            expName=v[0]
+        if(v[1]!=""):
+            runName=v[1]
+        if(v[2]!=""):
+            path=v[2]
+        
+        if(expName!= "" and path!= "" and runName!=""):
+            runDict[path]=[expName,runName]
         else:
-            if(seqRun==True and v[seqRunIndx]!=""):
-                runName=v[seqRunIndx]
-            if(exName==True and v[exNameIndx]!=""):
-                expName=v[exNameIndx]
-            if(filePath==True and v[filePathIndx]!=""):
-                path=v[filePathIndx]
-            
-            if(expName!= "" and path!= ""):
-                runDict[v[filePathIndx]]=[expName,runName]
-            
-            else:
-                messages.error(request,"something is not correct with your input file.")
+            messages.error(request,"something is not correct with your input file; check Experiment Name, Run Name and FilePath")
+            runDict.clear()
+            dupDict.clear()
+            break
+        
+        if(platform=="Nanopore"):
+            print("In Nanopore condition")
+            ftype=True
+            template_name='importNanopore.html'
+            orderList=nano
+            if(v[3]==""):
+                messages.error(request,"Missing Entry in Type column.")
+                runDict.clear()
+                dupDict.clear()
                 break
-            
-            if(md5sum==True and v[md5sumIndx]!=""):
-                md5Sum=v[md5sumIndx]
-                runDict[v[filePathIndx]].append(md5Sum)
-                context['md5sum']=True
             else:
-                context['md5sum']=False
-                
-                
-            if(sha256==True and v[sha256Indx]!=""):
-                sha256sum=v[sha256Indx]
-                runDict[v[filePathIndx]].append(sha256sum)
-                context['sha256sum']=True
-            else:
-                context['sha256sum']=False
+                filetype=v[3]
+                runDict[path].append(filetype) 
+
+
+        if(v[4]!=""):
+            md5Sum=v[4]
+            md5sum=True
+        runDict[path].append(md5Sum)           
+        if(v[5]!=""):
+            sha256sum=v[5]
+            sha256=True 
+        runDict[path].append(sha256sum)  
+        if(v[6]!=""):
+            BackupPath=v[6]
+            bpath=True
+        runDict[path].append(BackupPath)
             
-            if(path in exsisting_files):
-               dupDict[path]=runDict[path]
-               del runDict[path]
-        count+=1
+        if(path in exsisting_files):
+           dupDict[path]=runDict[path]
+           del runDict[path]
```

Comment:
please display a message here - that some of the files user is trying to add - already exist in the database ...